doc/configuration/Configuration.md

```diff
@@ -0,0 +1,319 @@
+
+
+### Email Settings
+___
+Settings to configure email signup, notifications, security, and SMTP options. 
+
+### Signup
+
+```"EnableSignUpWithEmail": true```
+*"true": Allow team creation and account signup using email and password; “false”: Email signup is disabled and users are not able to invite new members. This limits signup to single-sign-on services like OAuth or LDAP.*
+
+### Notifications
+
+```"SendEmailNotifications": false```
+*"true": Enables sending of email notifications. Developers may set this field to false to skip email setup for faster development.*
+
+```"RequireEmailVerification": false```
+*"true": Require email verification after account creation prior to allowing login; “false”: Users do not need to verify their email address prior to login. Developers may set this field to false so skip sending verification emails for faster development.*
+XXX: This feature is jenky with 
+
+
+```"FeedbackName": ""```
+*Name displayed on email account used when sending notification emails from Mattermost system.*
+
+```"FeedbackEmail": ""```
+*Email displayed on email account used when sending notification emails from Mattermost system.*
+
+### SMTP
+
+```"SMTPUsername": ""```
+*Obtain this credential from administrator setting up your email server.*
+
+```"SMTPPassword": ""```
+*Obtain this credential from administrator setting up your email server.*
+
+```"SMTPServer": ""```
+*Location of SMTP email server.*
+
+```"SMTPPort": ""```
+*Port of SMTP email server.*
+
+### Security
+
+```"ConnectionSecurity": ""```
+*"none": send email over an unsecure connection; "TLS": communication between Mattermost and your email server is encrypted; “STARTTLS”: attempts to upgrade an existing insecure connection to a secure connection using TLS.*
+
+```"InviteSalt": "bjlSR4QqkXFBr7TP4oDzlfZmcNuH9YoS"```
+*32-character (to be randomly generated via Admin Console) salt added to signing of email invites.*
+
+
+```"PasswordResetSalt": "vZ4DcKyVVRlKHHJpexcuXzojkE5PZ5eL"```
+*32-character (to be randomly generated via Admin Console) salt added to signing of password reset emails.*
+
+
+### File Settings
+___
+Settings to configure storage, appearance, and security of files and images.
+
+### File Storage
+
+```"DriverName": "local"```
+*System used for file storage. “local”: Files and images are stored on the local file system. “amazons3”: Files and images are stored on Amazon S3 based on the provided access key, bucket and region fields.*
+
+```"Directory": "./data/"```
+*Directory to which files are written. If blank, directory will be set to ./data/.*
+
+```"AmazonS3AccessKeyId": ""```
+*Obtain this credential from your Amazon EC2 administrator.*
+
+```"AmazonS3SecretAccessKey": ""```
+*Obtain this credential from your Amazon EC2 administrator.*
+
+```"AmazonS3Bucket": ""```
+*Name you selected for your S3 bucket in AWS.*
+
+```"AmazonS3Region": ""```
+*AWS region you selected for creating your S3 bucket.*
```

Comment:
Maybe link here http://docs.aws.amazon.com/general/latest/gr/rande.html#s3_region and let them know they need to pick from the `Region` column